src/Encamina.Enmarcha.SemanticKernel/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -15,26 +15,42 @@ namespace Microsoft.Extensions.DependencyInjection;
 public static class IServiceCollectionExtensions
 {
     /// <summary>
-    /// Adds and configures the <see cref="MemoryManager"/> type as transient service instance of the <see cref="IMemoryManager"/> service to the <see cref="IServiceCollection"/>.
+    /// Adds and configures the <see cref="MemoryStoreExtender"/> type as transient service instance of the <see cref="IMemoryStoreExtender"/> service to the <see cref="IServiceCollection"/>.
     /// </summary>
     /// <remarks>
-    /// This method registers the <see cref="MemoryManager"/> type as a transient service due to its dependency on <see cref="Kernel"/>, which is typically registered as transient or scoped (rarely as singleton).
+    /// This method registers the <see cref="MemoryStoreExtender"/> type as a transient service due to its dependency on <see cref="Kernel"/>, which is typically registered as transient or scoped (rarely as singleton).
     /// </remarks>
     /// <param name="services">The <see cref="IServiceCollection"/> to add services to.</param>
     /// <returns>The <see cref="IServiceCollection"/> so that additional calls can be chained.</returns>
+    [Obsolete("This extension is obsolete and will be removed in a future version. Use the Microsoft.Extensions.DependencyInjection.IServiceCollectionExtensions.AddMemoryStoreExtender() extension instead.", false)]
     public static IServiceCollection AddMemoryManager(this IServiceCollection services)
     {
         services.TryAddSingleton<IMemoryManager, MemoryManager>();
 
         return services;
     }
 
+    /// <summary>
+    /// Adds and configures the <see cref="MemoryStoreExtender"/> type as transient service instance of the <see cref="IMemoryStoreExtender"/> service to the <see cref="IServiceCollection"/>.
+    /// </summary>
+    /// <remarks>
+    /// This method registers the <see cref="MemoryStoreExtender"/> type as a transient service due to its dependency on <see cref="Kernel"/>, which is typically registered as transient or scoped (rarely as singleton).
```

Comment:
The type `MemoryStoreExtender` (currently) does not have a depdendency with the `Kernel` type. Maybe in the past this was true, but it is not the case anymore.

This could apply to other comments in the code.

Reply:
Also it says `Transient` but the code says `Singleton` which is the right lifecycle 😉

Reply:
I'm noticing that the (now) obsolete method `AddMemoryManager` has the comments wrong. Remember the «boy scout» principle, lets try to improve what we see is not really right 🙂

Reply:
Thank you for your contribution. You are right. I will update it.